### PR TITLE
Gun and Armour parts

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

1. Increases the material from gun and armour parts (gun part is 1.2 plasteel, armour part is 5 wood, steel, cardboard and plastic)
reduces gun part size to small
2. allows NT to print parts pouch
3. also makes the part pouch list look nicer

## Why It's Good For The Game

1. having the parts give only 1 sheet in materials made them highly useless to those who didn't want to craft anything with them, also having the gun parts not be the same size as a crowbar is nice
2. consistency, NT can already print almost every pouch
3. doesn't affect gameplay, just looks better and more consistent with other pouches

## Changelog
:cl:
add: Added parts pouch to a NT disk (the one with pouches and clothes)
tweak: tweaked materials from gun and armour parts, gun part is smaller
/:cl:

### CODE REVIEW

**Comment by Wouju** [comment]
> You need 15 gun parts to make a random gun, so that's 90 plasteel

bugger, gonna check something real quick then

**Comment by Wouju** [comment]
> You need 15 gun parts to make a random gun, so that's 90 plasteel
Gonna lower the cost then, to 1,2, so recycling 15 gun parts results in 18 plasteel (lowest cost of a gun is i believe 10 plasteel for the giskard, not counting novakovic since it doesn't use plasteel)
sounds good?

**Comment by TheShown911** [comment]
> > You need 15 gun parts to make a random gun, so that's 90 plasteel
> > Gonna lower the cost then, to 1,2, so recycling 15 gun parts results in 18 plasteel (lowest cost of a gun is i believe 10 plasteel for the giskard, not counting novakovic since it doesn't use plasteel)
> > sounds good?

yea

File: code/datums/craft/crafting_items.dm [hunk 1]
```diff
@@ -57,7 +58,7 @@
 /obj/item/craft_frame/guns
 	name = "gun assembly"
 	desc = "Add some weapon parts to complete this, use your knowledge of mechanics and create a gun."
-	matter = list(MATERIAL_PLASTEEL = 5)
+	matter = list(MATERIAL_PLASTEEL = 6)
```
Comment (Evankhell561) [inline code/datums/craft/crafting_items.dm:61]: you need 5 of plasteel to create one of these.

File: code/datums/autolathe/biomatter.dm [hunk 2]
```diff
@@ -196,6 +196,10 @@
 	name= "Vial pouch"
 	build_path = /obj/item/weapon/storage/pouch/tubular/vial
 
+/datum/design/bioprinter/part
+	build_type = "Part pouch"
```
Comment (Firefox13) [inline code/datums/autolathe/biomatter.dm:200]:
```suggestion
	name = "Part pouch"
```

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**